shape.lic

```diff
@@ -46,108 +42,70 @@ class Shape
     args = parse_args(arg_definitions)
 
     @finish = args.finish
-    @chapter = args.chapter
-    @recipe_name = args.recipe_name
+    @enhancement = Hash[args.laminate.nil? => 'bow lamination', args.lighten.nil? => 'bow lightening', args.cable.nil? => 'bow cable-backing']
+    @recipe_name = args.recipe_name == nil ? (@enhancement[false]) : args.recipe_name
```

Comment:
These two lines were especially tough to follow for me. Can we simplify these, avoid creating an instance variable when we're using it just to assist creation of another variable on the next line, and avoid the double-negative-ish nature of `(@enhancement[false])`?